pages/common/git-push.md

```diff
@@ -1,23 +1,31 @@
 # git push
 
-> Push commits to a remote repository.
+> Push commits to a remote (upstream) repository.
```

Comment:
Just wondering, is there is use of adding "upstream" here ? Doesn't remote always mean upstream ?